### ISSUE TITLE
Getting SSL Error in downloading "distilroberta-base-paraphrase-v1" model embeddings

### ISSUE THREAD

**Issue by rahuliitkgp31** [UKPLab/sentence-transformers#571]
I am using google collab with PyTorch version 1.7.0+cu101
I am getting an SSL Error when I am trying to download "distilroberta-base-paraphrase-v1" model.

**Code**
from sentence_transformers import SentenceTransformer
model = SentenceTransformer('distilroberta-base-paraphrase-v1')

**Error**

SSLError                                  Traceback (most recent call last)
/usr/local/lib/python3.6/dist-packages/urllib3/connectionpool.py in urlopen(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)
    599                                                   body=body, headers=headers,
--> 600                                                   chunked=chunked)
    601 

24 frames
SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)

During handling of the above exception, another exception occurred:

MaxRetryError                             Traceback (most recent call last)
MaxRetryError: HTTPSConnectionPool(host='public.ukp.informatik.tu-darmstadt.de', port=443): Max retries exceeded with url: /reimers/sentence-transformers/v0.2/distilroberta-base-paraphrase-v1.zip (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)'),))

During handling of the above exception, another exception occurred:

SSLError                                  Traceback (most recent call last)
SSLError: HTTPSConnectionPool(host='public.ukp.informatik.tu-darmstadt.de', port=443): Max retries exceeded with url: /reimers/sentence-transformers/v0.2/distilroberta-base-paraphrase-v1.zip (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)'),))

During handling of the above exception, another exception occurred:

FileNotFoundError                         Traceback (most recent call last)
/usr/lib/python3.6/shutil.py in rmtree(path, ignore_errors, onerror)
    473         # lstat()/open()/fstat() trick.
    474         try:
--> 475             orig_st = os.lstat(path)
    476         except Exception:
    477             onerror(os.lstat, path, sys.exc_info())

FileNotFoundError: [Errno 2] No such file or directory: '/root/.cache/torch/sentence_transformers/sbert.net_models_distilroberta-base-paraphrase-v1'

**Comment by shamusfield-expel** [comment]
I'm also seeing this, exact same error for `SentenceTransformer('bert-base-nli-stsb-mean-tokens')`

**Comment by nreimers** [comment]
You can try to download the models from here and unzip them (and load from local disc then):
https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transformers/v0.2/

**Comment by rahuliitkgp31** [comment]
@nreimers Yea I manually downloaded the model and it is worked for me. Thanks

**Comment by anu43** [comment]
> You can try to download the models from here and unzip them (and load from local disc then):
> https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transformers/v0.2/

I was working with another model on google colab and don't have a fast internet connection to download and upload every time. When do you think that it will be solved?

**Comment by nreimers** [comment]
Not sure what the issue is with google colab, so sadly cannot say when it will be solved.

But you can also use colab to download files:
```
!wget https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transformers/v0.2/distilbert-base-nli-stsb-mean-tokens.zip
```

And the unzip with colab

**Comment by apostolosT** [comment]
I 've solved this issue in colab by doing the following:
`! wget --no-check-certificate https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transformers/v0.2/bert-large-nli-stsb-mean-tokens.zip`
`!unzip /content/bert-large-nli-stsb-mean-tokens.zip -d /root/.cache/torch/sentence_transformers/sbert.net_models_bert-large-nli-stsb-mean-tokens`

**Comment by nreimers** [comment]
I temporally switched the model download to a different server (to a mirror server). It should hopefully work.

I am not sure if the mirror server has all models from the main server. If some model is missing, let me know and I can upload it.

It appears that the authority that signed the SSL certificate (DFN) is somehow not recognized by all systems as legitimate.

**Comment by pritamdeka** [comment]
Hi @nreimers ,
The following models are also giving an error in colab:
`distilroberta-base-paraphrase-v1` and `xlm-r-distilroberta-base-paraphrase-v1`

**Comment by tianzongw** [comment]
> I 've solved this issue in colab by doing the following:
> `! wget --no-check-certificate https://public.ukp.informatik.tu-darmstadt.de/reimers/sentence-transformers/v0.2/bert-large-nli-stsb-mean-tokens.zip`
> `!unzip /content/bert-large-nli-stsb-mean-tokens.zip -d /root/.cache/torch/sentence_transformers/sbert.net_models_bert-large-nli-stsb-mean-tokens`

May I ask where to unzip for different models? tried roberta-large-nli-stsb-mean-tokens by simply replacing model names but failed in the upzip part.

**Comment by nreimers** [comment]
@pritamdeka 
Thanks, should be fixed  now.

You have to ensure that '/root/.cache/torch/sentence_transformers/' exists as path. Maybe you have to create it before you can run unzip

**Comment by pritamdeka** [comment]
@tianzongw You need to install sentence-transformers first before unzipping the pre-trained models using the method @apostolosT mentioned. Hopefully it would solve your error.

**Comment by nreimers** [comment]
The issue should be fixed now

**Comment by aimanian** [comment]
Hello, I installed the sentence-transformers in python, and tried to run the model = SentenceTransformer('paraphrase-MiniLM-L6-v2') in Jupyter notebook. None of the available models worked. I downloaded and unzipped the models in sentence-transformers main installed folder but I get license error as bellow. I appreciate your help.

HTTPSConnectionPool(host='huggingface.co', port=443): Max retries exceeded with url: /api/models/sentence-transformers/paraphrase-MiniLM-L6-v2 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1131)')))
